golden_response/staging/get_pop_obs/moscow.json

```diff
@@ -5248,158 +5252,158 @@
         }
       ]
     },
-    "dc/p/g54457nw57sm2": {
+    "dc/p/g5z6vl38vk2e": {
       "observations": [
         {
-          "id": "dc/o/zqgyls1pbtz2b",
+          "id": "dc/o/zsrndywlstt8d",
           "type": "Observation"
         },
         {
-          "id": "dc/o/zfl3yslnbmpt4",
+          "id": "dc/o/zdtd1spk5gwv2",
           "type": "Observation"
         },
         {
-          "id": "dc/o/zb52q7jngek6b",
+          "id": "dc/o/z7z407xg31hy2",
           "type": "Observation"
         },
         {
-          "id": "dc/o/x1sr127lk7jwb",
+          "id": "dc/o/xbf13lybllsxc",
           "type": "Observation"
         },
         {
-          "id": "dc/o/w1fn385s6gsg8",
+          "id": "dc/o/x471rtdcjzfkg",
           "type": "Observation"
         },
         {
-          "id": "dc/o/sm52wtrmd146",
+          "id": "dc/o/vv56ly9v4017d",
           "type": "Observation"
         },
         {
-          "id": "dc/o/rc50449g6mqpf",
+          "id": "dc/o/qshvv1nwyq6td",
           "type": "Observation"
         },
         {
-          "id": "dc/o/n9l25c68b32b2",
+          "id": "dc/o/q7qk8nzt5lf8d",
           "type": "Observation"
         },
         {
-          "id": "dc/o/lcnjrxv395713",
+          "id": "dc/o/mgcmm247fwe95",
           "type": "Observation"
         },
         {
-          "id": "dc/o/jbpnd7zcv62r8",
+          "id": "dc/o/lxtg6qw9s86e2",
           "type": "Observation"
         },
         {
-          "id": "dc/o/j4hvvzr0sn8s9",
+          "id": "dc/o/jrmbt8k10pdch",
           "type": "Observation"
         },
         {
-          "id": "dc/o/gdqyq0nnkt27d",
+          "id": "dc/o/gry5np8wncqtb",
           "type": "Observation"
         },
         {
-          "id": "dc/o/e13xmyyrlnzjd",
+          "id": "dc/o/fh771rsy8sdy",
           "type": "Observation"
         },
         {
-          "id": "dc/o/ddprf6mx7lj4g",
+          "id": "dc/o/eb19xzgw3h8kf",
           "type": "Observation"
         },
         {
-          "id": "dc/o/cp5z6w99y9x7c",
+          "id": "dc/o/cd6dvyt0lwwe7",
           "type": "Observation"
         },
         {
-          "id": "dc/o/bcby5dhj9819f",
+          "id": "dc/o/81m9eqeglx94c",
           "type": "Observation"
         },
         {
-          "id": "dc/o/5mqvttb7cc6cg",
+          "id": "dc/o/65819nqwvhs95",
           "type": "Observation"
         },
         {
-          "id": "dc/o/2rhnrqrktbem7",
+          "id": "dc/o/22rd92cxb0k67",
+          "type": "Observation"
+        },
+        {
+          "id": "dc/o/1g04ydkpt2rjb",
           "type": "Observation"
         }
       ]
     },
-    "dc/p/g5z6vl38vk2e": {
+    "dc/p/h5wfbkjpcfpx4": {
```

Comment:
Also OECD change.

Reply:
Does "prod definitely needs updating for Moscow total pop" mean because of OECD changes the population stat  attached to moscow has changed?

Reply:
Argh, I think I made a mistake. The previous pop was actually for Males of a certain age, which explains why the number is so low. And it actually only moved in the file. The current population is new, though.

Reply:
Why is the current population "new"? Did the resolution change affect it?